bears/natural_language/ProseLintBear.py

```diff
@@ -14,6 +14,7 @@ class ProseLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Spelling', 'Syntax', 'Formatting', 'Grammar'}
```

Comment:
i guess grammar is regarding ``i walk`` instead of ``i walks`` and syntax is probably not having ``i wal,sa7s``?